src/js/views/patients/worklist/worklist_views.js

```diff
@@ -179,10 +179,16 @@ const ListView = CollectionView.extend({
   },
   childViewTriggers: {
     'render': 'listItem:render',
+    'click:patientSidebarButton': 'patientSidebarButton:click',
```

Comment:
So this is already bubbling up this event... so either this'd be better as a `childViewEvent` if it s meant to be handled by this view, or the app can do what `onPatientSidebarButtonClick` is doing, which I think in this case is probably preferred.

the app can just listen to the click.  at the app level destructing the view is also nice ie: `onPatientSidebarButtonClick({ model })`